### PR TITLE
Separate script to acquire SDK, limit CI

### PR DESCRIPTION
**Pull request by dg0yt**
- Separate script. Can be run manually on linux.
- Add more packages as documented on https://doc.qt.io/qt-6/android-getting-started.html#android-sdk-mandatory-packages
- Limit CI to android (as long as developing #35845)

### CODE REVIEW

File: scripts/azure-pipelines/acquire-android-sdk.ps1 [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Write-Host "Downloading the Android SDK"
+& "./vcpkg" x-download sdk-commandlinetools-linux-$ToolsVersion.zip "--sha512=64b7d18ee7adeb1204eaa2978091e874dc9af9604796b64e1a185a11c15325657383fc9900e55e4590c8b8a2784b3881745d2f32daef1207e746c0ee41c2b72b" "--url=https://dl.google.com/android/repository/commandlinetools-linux-${ToolsVersion}.zip"
+
+$env:ANDROID_HOME = Join-Path $Pwd "android-sdk"
+Write-Host "Setting up the Android SDK in $env:ANDROID_HOME"
+& unzip -q sdk-commandlinetools-linux-$ToolsVersion.zip -d android-sdk
+# https://doc.qt.io/qt-6/android-getting-started.html
+& bash -c 'yes | ./android-sdk/cmdline-tools/bin/sdkmanager "--sdk_root=$ANDROID_HOME" "platform-tools" "platforms;android-34" "build-tools;34.0.0"'
+& ./android-sdk/cmdline-tools/bin/sdkmanager "--sdk_root=$ANDROID_HOME" --list_installed
```
Comment (dg0yt) [inline scripts/azure-pipelines/acquire-android-sdk.ps1]: :facepalm:

**Comment by dg0yt** [comment]
~~~
Installed packages:
  Path                 | Version | Description                | Location            
  -------              | ------- | -------                    | -------             
  build-tools;34.0.0   | 34.0.0  | Android SDK Build-Tools 34 | build-tools/34.0.0  
  platform-tools       | 35.0.1  | Android SDK Platform-Tools | platform-tools      
  platforms;android-34 | 3       | Android SDK Platform 34    | platforms/android-34
~~~

**Comment by dg0yt** [comment]
Ping @m-kuhn

**Comment by m-kuhn** [comment]
Good move, thanks

**Pull request merged**